### ISSUE TITLE
pyclean3 failing on Windows 10

### ISSUE THREAD

**Issue by HoECoder** [bittner/pyclean#10]
I get the following error while attempting to run py3clean on Windows 10 with a fresh install of Python 3.6:

```
py3clean --help
Traceback (most recent call last):
  File "C:\Python\Python36\Scripts\py3clean-script.py", line 11, in <module>
    load_entry_point('pyclean==1.0.0', 'console_scripts', 'py3clean')()
  File "C:\Python\Python36\lib\site-packages\pkg_resources\__init__.py", line 490, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "C:\Python\Python36\lib\site-packages\pkg_resources\__init__.py", line 2859, in load_entry_point
    return ep.load()
  File "C:\Python\Python36\lib\site-packages\pkg_resources\__init__.py", line 2450, in load
    return self.resolve()
  File "C:\Python\Python36\lib\site-packages\pkg_resources\__init__.py", line 2456, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "C:\Python\Python36\lib\site-packages\pyclean\py3clean.py", line 32, in <module>
    from debpython import files as dpf
ModuleNotFoundError: No module named 'debpython'
```

It appears that the script is still relying on a Debian only python package `debpython`. See [Line #27 of py3clean](https://github.com/bittner/pyclean/blob/master/pyclean/py3clean.py#L27).

Also, there are hard-coded paths ([Line 25 of py3clean.py](https://github.com/bittner/pyclean/blob/master/pyclean/py3clean.py#L25)) to python binaries that are:
- Linux specific 
- Possibly distribution specific

Is there any openness to working on this without resorting to `debpython` and fixing up the hard-coded paths? This looks like a great utility, and the write up really had me wanting to use it, why reinvent the wheel.

**Comment by bittner** [comment]
Thank you for your interest in PyClean!

Yes, that's true, the way PyClean originally started is by pulling together the existing implementations from the Debian project (for Python 2, Python 3 and PyPy), and moving beyond this has not yet borne fruit.

The main reason for this is that I'm cautious to not break backward-compatibility, so that PyClean gets a chance to be used by the Debian project, replacing today's existing implementations. And writing tests that ensure the existing implementation has proven difficult. -- In other words, I don't have a full picture of what the three scripts do as a whole. Sad story. :smirk:

Having said that, I'm really open to any contributions that bring us forward, and I'm willing to sacrifice legacy-implementation compatibility for that provided that the actual cleaning of byte code is still performed correctly. I'm also willing to do pair-programming if you're interested.

**Comment by bittner** [comment]
Related to #8.

**Comment by HoECoder** [comment]
Sure thing. I've got a quick-and-dirty I put together based on this. Let me fork and submit a PR in the next few days and we'll go from there!

Stay safe, stay healthy!

**Comment by HoECoder** [comment]
Just opened #11 to start addressing it.

**Comment by rittneje** [comment]
+1 for this issue. The python:3-slim Docker image does not include the debpython module, so this script does not work unless you use the "full" image.

**Comment by bittner** [comment]
I have released v2.0.0, which should now work on any platform supported by Python3.

@HoECoder @rittneje Can you try whether it does what you expect?

```console
C:\your\code\folder> python3 -m pip install --upgrade pyclean
```
```console
C:\your\code\folder> pyclean --version
2.0.0
```
```console
C:\your\code\folder> pyclean .
```
You can also use the `--dry-run` and  `--verbose` options to help you understand what the command is doing.

See the usage screen for details:
```console
C:\your\code\folder> pyclean --help
```

**Comment by bittner** [comment]
Version 2.0.0 should now work flawlessly even on Windows and macOS. Let me know if you see any issues.
